tests/integration/common/s2n_test_scenario.py

```diff
@@ -103,6 +103,9 @@ def get_libcrypto():
 
 ALL_CURVES = ["P-256", "P-384"]
 
+# This will only work until we support X25519. Then we have to pick a new unsupported curve.
+UNSUPPORTED_CURVES = ['X25519']
+
 
```

Comment:
we are going to re-activate x25519. Instead, please use x448(0x001E)

Reply:
Sounds good, I'll update to use that curve.